docker/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -euo pipefail
```

Comment:
This is a bash-ism and may not work as expected with Alpine sh (I don't know if Alpine sh is busybox or bash, but you can make it use `bash` explicitly to make sure).

Reply:
fixed in https://github.com/databricks/cli/pull/1369